x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -191,7 +191,7 @@ const alertingAuthorizationFilterOpts: AlertingAuthorizationFilterOpts = {
   type: AlertingAuthorizationFilterType.KQL,
   fieldNames: { ruleTypeId: 'alert.attributes.alertTypeId', consumer: 'alert.attributes.consumer' },
 };
-export class AlertsClient {
+export class RulesClient {
```

Comment:
What do you think about renaming the variables in this file as part of this terminology renaming? example `alertTypeRegistry` -> `ruleTypeRegistry`?

Reply:
I noticed that as well - and I'd guess there are probably more lurking "alerts" changes to be made in the code.  

Making that a follow-up issue would be fine to me.

Reply:
A follow up issue sounds good :)

Reply:
I think this is a good idea. Probably the changed files won't increase much :-)